Fabric.Authorization.AccessControl/src/app/services/fabric-auth-edwadmin.service.ts

```diff
@@ -52,4 +53,22 @@ export class FabricAuthEdwAdminService extends FabricBaseService {
         .replace('{groupName}', groupName)
     );
   }
+
+  private setIdPAndTenant(url: string, identityProvider: string, tenantId: string) {
+    url = this.setQueryParameters(url, 'identityProvider', identityProvider);
+    url = this.setQueryParameters(url, 'tenantId', tenantId);
+    return url;
+  }
+
+  private setQueryParameters(url: string, key: string, val: string) {
```

Comment:
Is there an easier way of building up a url with query params in angular than manually parsing the string?